langfun/core/llms/google_genai.py

```diff
@@ -307,6 +307,17 @@ def get(
 # Public Gemini models.
 #
 
+class Gemini2_0_Flash_Exp(GenAI):  # pylint: disable=invalid-name
```

Comment:
Consider `GeminiFlash2_0_Exp` to be consistent with existing models (e.g. GeminiFlash1_5).